quote.ftd

```diff
@@ -2450,6 +2491,201 @@ border-bottom-left-radius.px: 8
 
 
 
+-- component background-image:
+optional caption title:
+body body:
+ftd.image-src quote-background:
+string text-color:
+
+-- ftd.column:
+width: fill-container
+background.image: $background-image.quote-background
+color: $background-image.text-color
+
+
+-- ftd.desktop:
+
+-- quote-7-desktop: $background-image.title
+body: $background-image.body
+
+-- end: ftd.desktop
+
+-- ftd.mobile:
+
+-- quote-7-mobile: $background-image.title
+body: $background-image.body
+
+-- end: ftd.mobile
+
+-- end: ftd.column
+
+-- end: background-image
+
+
+
+
+
+
+
+
+
+-- component quote-7-desktop:
+optional caption title:
+body body:
+ftd.image-src logo: $assets.files.static.quote.svg
+
+-- ftd.column:
+padding-vertical.px: 32
+padding-horizontal.px: 32
+width: fill-container
+max-width.fixed.px: 800
+align-self: center
+margin-top.px: 24
+margin-bottom.px: 24
+
+-- ftd.column:
+border-top-width.px: 2
+border-color: $inherited.colors.text
+anchor: parent
+top.px: 38
+left.px: 148
+width.fixed.percent: 60
+align-self: center
+
+-- end: ftd.column
+
+-- ftd.column:
+border-bottom-width.px: 2
+border-color: $inherited.colors.text
+anchor: parent
+right.px: 146
+bottom.px: 38
+width.fixed.percent: 60
+
+-- end: ftd.column
+
+-- ftd.column:
+align-self: center
+width.fixed.percent: 60
+padding-vertical.px: 32
+max-height.fixed.px: 400
+
+-- ftd.text: 
+role: $inherited.types.copy-large
+
+
+$quote-7-desktop.body
+
+-- ftd.row:
+align-self: center
+spacing.fixed.px: 8
+
+-- ftd.text: ~
+role: $inherited.types.copy-regular
+
+-- ftd.text: $quote-7-desktop.title
+role: $inherited.types.copy-regular
+
+
+-- end: ftd.row
```

Comment:
```suggestion
-- ftd.row:
align-self: center
spacing.fixed.px: 8
role: $inherited.types.copy-regular

-- ftd.text: ~

-- ftd.text: $quote-7-desktop.title

-- end: ftd.row
```

Reply:
Hi @Arpita-Jaiswal , Thank you for the review!! I have made the changes.